flip_tdi/helpers/ftdi_usb.c

```diff
@@ -495,7 +489,7 @@ FtdiUsb* ftdi_usb_start(void) {
     ftdi_usb->usb.cfg_descr = (void*)&usb_ftdi_cfg_descr;
 
     if(!furi_hal_usb_set_config(&ftdi_usb->usb, ftdi_usb)) {
-        FURI_LOG_E(TAG, "USB locked, cannot start Mass Storage");
+        FURI_LOG_E(TAG, "USB locked, cannot start FlipTDI");
```

Comment:
Problem is still there: 
- start qflipper
- open app
- try to exit 

Also if USB is locked we show notification and then exit app. See GPIO app for more details